awscli/clidocs.py

```diff
@@ -48,6 +48,8 @@ def _get_argument_type_name(self, shape, default):
             return 'JSON'
         if is_document_type(shape):
             return 'document'
+        if is_streaming_blob_type(shape):
```

Comment:
As a rule of thumb, for any new logic we add there should be a test for it. So we should make sure we have unit tests for adding this new type to documentation and the note on the value to specify for a streaming blob. We probably can add it somewhere in `tests/unit/test_clidocs.py`.